llvm/include/llvm/MC/MCInstrDesc.h

```diff
@@ -203,8 +203,8 @@ class MCInstrDesc {
   // find other information elsewhere in the same table.
 
   unsigned short Opcode;         // The opcode number
-  unsigned short NumOperands;    // Num of args (may be more if variable_ops)
-  unsigned char NumDefs;         // Num of args that are definitions
+  uint32_t NumOperands;          // Num of args (may be more if variable_ops)
```

Comment:
This is file is not related to SelectionDAG. It only affects assembly instructions defined for a target. There should be checks in tablegen that these sizes are enough.